unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -102,9 +102,11 @@ def execute_query(
         table_name = str(kwargs.get("table_name"))
         try:
             if sql_values:
-                engine.query(sql_query, job_config=sql_values)
+                query_job = engine.query(sql_query, job_config=sql_values)
+                query_job.result()
             else:
-                engine.query(sql_query)
+                query_job = engine.query(sql_query)
+                query_job.result()
```

Comment:
@kirtimanmishrazipstack NIT: Can you move `query_job.result()` outside the if-else statement? since it is being called in both the cases.

Reply:
done